transforms:
  - parent_frame: "base_link"
    child_frame: "lidar_link"
    x: 0.0
    y: 0.0
    z: 0.0
    roll: 0.0
    pitch: 0.0
    yaw: 0.0

  - parent_frame: "base_link"
    child_frame: "imu_link"
    x: 0.0
    y: 0.07
    z: 0.0
    roll: 0.0
    pitch: 0.0
    yaw: 0.0

  - parent_frame: "imu_link"
    child_frame: "id001_base_link"
    x: 0.0
    y: 0.0
    z: 0.0
    roll: 0.0
    pitch: 0.0
    yaw: 0.0